### DISCUSSION TITLE
Checkout Spec

### DISCUSSION THREAD

**Discussion by dtdionne** [solidusio/solidus#5375] [Troubleshooting]
Greetings, newb here so for starters, forgive me...

With a totally fresh "starter/stripe" install should this work?

$ rspec ./spec/system/frontend/solidus_stripe/checkout_spec.rb

**Comment by elia** [comment]
It should, the most common issue we've seen so far is missing Stripe's testing API/SECRET keys, see this section for how to setup them in case you didn't: https://github.com/solidusio/solidus_stripe#development
